packages/ui/src/components/KafkaInstanceDrawer/components/KafkaConnectionTabP2.tsx

```diff
@@ -287,7 +287,7 @@ export const KafkaConnectionTabP2: FunctionComponent<
           {t("connection-tab:sasl_oauthbearer_description")}
         </Text>
         <strong>{t("connection-tab:token_endpoint_url")}</strong>
-        {isKafkaPending ? (
+        {isKafkaPending || tokenEndPointUrl === undefined ? (
```

Comment:
```suggestion
        {tokenEndPointUrl === undefined ? (
```
I don't think we need to wait for the instance to be ready as this url pointed to SSO.